File: NetFramework/XHtmlEditor/CS/RichTextEditor/Tabs/MailingsTab.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Windows.Forms;
using System.Drawing;
using C1.Win.Ribbon;

namespace RichTextEditor
{
    class MailingsTab : ExtendedC1RibbonTab
    {
        public MailingsTab()
        {
            this.ID = this.Text = "Mailings";
            Groups.Add(new RibbonGroup("Create"));
            Groups.Add(new RibbonGroup("Start Mail Merge"));
            Groups.Add(new RibbonGroup("Write && Insert Fields"));
            Groups.Add(new RibbonGroup("Preview Results"));
            Groups.Add(new RibbonGroup("Finish"));
        }
    }
}

File: NetFramework/XHtmlEditor/VB/C1WordsX/Tabs/MailingsTab.vb
Imports System.Collections.Generic
Imports System.Text
Imports System.Windows.Forms
Imports System.Drawing
Imports C1.Win.C1Ribbon

Namespace C1WordsX
	Class MailingsTab
		Inherits C1TextEditorRibbonTab
		Public Sub New()
			Me.ID = InlineAssignHelper(Me.Text, "Mailings")
			Groups.Add(New RibbonGroup("Create"))
			Groups.Add(New RibbonGroup("Start Mail Merge"))
			Groups.Add(New RibbonGroup("Write && Insert Fields"))
			Groups.Add(New RibbonGroup("Preview Results"))
			Groups.Add(New RibbonGroup("Finish"))
		End Sub
		Private Shared Function InlineAssignHelper(Of T)(ByRef target As T, value As T) As T
			target = value
			Return value
		End Function
	End Class
End Namespace
